packages/swirl-components/src/components/swirl-date-picker/swirl-date-picker.css

```diff
@@ -292,3 +292,21 @@
     font-style: normal;
   }
 }
+
+.date-picker__date--disabled {
+  color: var(--s-text-disabled);
+  cursor: default;
+  pointer-events: none;
```

Comment:
I found that, even if the date cell is disable, we can click it and navigate to the previous or next month.

From a user perspective I wouldn't expect that to happen, and as a quick fix I disabled `pointer-events`.

It's not a solution I like, to be fair. What do you think regarding UX and technical implementation?

Reply:
Sounds like an issue with the wc-datepicker lib. Workaround is fine.

Reply:
Agreed. I will try to keep in mind to removed it when we do an update of wc-datepicker lib.